### ISSUE TITLE
Add a new FolderColorRgb option to both New-GSDriveFile and Update-GSDriveFile

### ISSUE THREAD

**Issue by WJurecki** [scrthq/PSGSuite#173]
Currently unable to set drive folder colors using PSGSuite.

Would like to be able to assign colors to drive folders.

New parameter could be added, such as:
```
        [parameter(Mandatory = $false)]       
        [ValidateSet('ChocolateIceCream','OldBrickRed','Cardinal','WildStrawberries','MarsOrange','YellowCab','Spearmint','VernFern','Asparagus','SlimeGreen','DesertSand','Macaroni','SeaFoam','Pool','Denim','RainySky','BlueVelvet','PurpleDino','Mouse','MountainGrey','Earthworm','BubbleGum','PurpleRain','ToyEggplant')]
        [string]
        $FolderColorRgb
```

with a color dictionary of UI allowed colors being defined as:

```
$ColorDictionary = @{
            ChocolateIceCream    = '#ac725e'
            OldBrickRed          = '#d06b64'
            Cardinal             = '#f83a22'
            WildStrawberries     = '#fa573c'
            MarsOrange           = '#ff7537'
            YellowCab            = '#ffad46'
            Spearmint            = '#42d692'
            VernFern             = '#16a765'
            Asparagus            = '#7bd148'
            SlimeGreen           = '#b3dc6c'
            DesertSand           = '#fbe983'
            Macaroni             = '#fad165'
            SeaFoam              = '#92e1c0'
            Pool                 = '#9fe1e7'
            Denim                = '#9fc6e7'
            RainySky             = '#4986e7'
            BlueVelvet           = '#9a9cff'
            PurpleDino           = '#b99aff'
            Mouse                = '#8f8f8f'
            MountainGrey         = '#cabdbf'
            Earthworm            = '#cca6ac'
            BubbleGum            = '#f691b2'
            PurpleRain           = '#cd74e6'
            ToyEggplant          = '#a47ae2'
        };
```

filling the body of the API call with 

```
            if ($FolderColorRgb) {
                $body.FolderColorRgb = $ColorDictionary[$FolderColorRgb];
            } 
```

I believe this will work for Drive folders.  I'm not sure if this will cause any issues if attempted on a non-folder file or Team Drive file.

This should allow automation of the UI feature to change a folder's color as seen here:

![image](https://user-images.githubusercontent.com/48518449/55557393-bcea8200-56b7-11e9-9651-11d26bd4ad0d.png)

**Comment by scrthq** [comment]
Hey @WJurecki - Great suggestion! I'm on it ASAP, although PR's always welcome =]

**Comment by scrthq** [comment]
Should be getting this sorted by tonight!

**Comment by scrthq** [comment]
available now in v2.26.1!
